library/src/main/java/com/bumptech/glide/manager/RequestManagerRetriever.java

```diff
@@ -60,6 +71,11 @@
    */
   private final Handler handler;
 
+  // Objects used to find Fragments and Activities containing views.
+  private final ArrayMap<View, Fragment> tempViewToSupportFragment = new ArrayMap<>();
+  private final ArrayMap<View, android.app.Fragment> tempViewToFragment = new ArrayMap<>();
```

Comment:
I'm not sure, but I bumped the min sdk version to 14, so this should be a non-issue.